src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/set_media_clock_timer_test.cc

```diff
@@ -358,6 +358,114 @@ TEST_F(SetMediaClockRequestTest, OnEvent_Canceled) {
   command->on_event(event);
 }
 
+TEST_F(SetMediaClockRequestTest, MessageWithForwardSeekIndicator_TRACK) {
+  MessageSharedPtr msg = CreateMsgParams();
+  mobile_apis::SeekIndicatorType::eType seek_indicator_type =
+      mobile_apis::SeekIndicatorType::TRACK;
+
+  (*msg)[am::strings::msg_params][am::strings::forward_seek_indicator] =
+      seek_indicator_type;
+  std::shared_ptr<SetMediaClockRequest> command(
+      CreateCommand<SetMediaClockRequest>(msg));
+
+  ExpectationsSetupHelper(true);
+
+  command->Run();
+
+  EXPECT_EQ(
+      seek_indicator_type,
+      static_cast<mobile_apis::SeekIndicatorType::eType>(
+          (*msg)[am::strings::msg_params][am::strings::forward_seek_indicator]
+              .asInt()));
+  auto msg_params = (*msg)[am::strings::msg_params];
```

Comment:
Maybe it's better to move this line before first usage of `(*msg)[am::strings::msg_params]`